src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Button() {
+  return (
+          <button className="btn btn-primary">Search IDs</button>
```

Comment:
This component is not very reusable right now - if we tried to use it elsewhere in our app then it would always say "Search IDs". How could we change this component to be more reusable?